### ISSUE TITLE
Android version doesn't work

### ISSUE THREAD

**Issue by micahlt** [WorldLanguages/ScratchNotifier.cf#22]
I've used ScratchNotifier on Android for over a year, but now it won't work!  When I go to the page and enter my username it just asks me again, over and over.

**Comment by micahlt** [comment]
Sure!  Out of curiosity, what was the issue?

**Comment by micahlt** [comment]
@WorldLanguages Once again, it appears to have stopped working on Android!  Notifications are turned on in Chrome.  I’m running Android Marshmallow and the latest version of Chrome.

**Comment by WorldLanguages** [comment]
@micahlt, do you remember when it last worked?

**Comment by micahlt** [comment]
It last worked just now!  Randomly!  It hasn't worked for about 2 months!

**Comment by WorldLanguages** [comment]
Yeah, and no one told me about it, I guess no one uses the mobile version, specially because of Apple devices not supporting web push
